Feature storagestate

mlohvynenko opened the pull request.

mykola-kobets-epam reviewed (approved):
`Reviewed-by: Mykola Kobets <mykola_kobets@epam.com>`

MykolaSuperman reviewed (approved):
`Reviewed-by: Mykola Solianko <mykola_solianko@epam.com>`

mlohvynenko committed:
common: utils: move change owner function to fs platform

This patch moves change owner function to fs platform to reflect
core lib interface changes.

Signed-off-by: Mykhailo Lohvynenko <mykhailo_lohvynenko@epam.com>
Reviewed-by: Mykola Kobets <mykola_kobets@epam.com>
Reviewed-by: Mykola Solianko <mykola_solianko@epam.com>
Reviewed-by: Oleksandr Grytsov <oleksandr_grytsov@epam.com>

mlohvynenko committed:
sm: image: use fs platform to change owner

This patch updates imagehandler to use fs platform interface
to change owner of the image file.

Signed-off-by: Mykhailo Lohvynenko <mykhailo_lohvynenko@epam.com>
Reviewed-by: Mykola Kobets <mykola_kobets@epam.com>
Reviewed-by: Mykola Solianko <mykola_solianko@epam.com>
Reviewed-by: Oleksandr Grytsov <oleksandr_grytsov@epam.com>

mlohvynenko committed:
cm: storagestate: remove storage state implementation

This patch removes the storage state module implementation.
The storage state module is implemented in the core library.

Signed-off-by: Mykhailo Lohvynenko <mykhailo_lohvynenko@epam.com>
Reviewed-by: Mykola Kobets <mykola_kobets@epam.com>
Reviewed-by: Mykola Solianko <mykola_solianko@epam.com>
Reviewed-by: Oleksandr Grytsov <oleksandr_grytsov@epam.com>

mlohvynenko committed:
common: utils: fswatcher: implement core lib interface

This patch updates the fswatcher implemntation to implement
the core library interface.

Signed-off-by: Mykhailo Lohvynenko <mykhailo_lohvynenko@epam.com>
Reviewed-by: Mykola Kobets <mykola_kobets@epam.com>
Reviewed-by: Mykola Solianko <mykola_solianko@epam.com>
Reviewed-by: Oleksandr Grytsov <oleksandr_grytsov@epam.com>

mlohvynenko committed:
all: add missing fs header include

Signed-off-by: Mykhailo Lohvynenko <mykhailo_lohvynenko@epam.com>
Reviewed-by: Mykola Kobets <mykola_kobets@epam.com>
Reviewed-by: Mykola Solianko <mykola_solianko@epam.com>
Reviewed-by: Oleksandr Grytsov <oleksandr_grytsov@epam.com>

al1img reviewed (approved):
`Reviewed-by: Oleksandr Grytsov <oleksandr_grytsov@epam.com>`

mlohvynenko committed:
aoscoreconfig: rename file path len constant

This constant was move to common from fs in the core library.

Signed-off-by: Mykhailo Lohvynenko <mykhailo_lohvynenko@epam.com>
Reviewed-by: Mykola Kobets <mykola_kobets@epam.com>
Reviewed-by: Mykola Solianko <mykola_solianko@epam.com>
Reviewed-by: Oleksandr Grytsov <oleksandr_grytsov@epam.com>
